docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3"
+
+services:
+  app:
+    restart: always
```

Comment:
Not an issue with this, but still I hope you do understand the difference between different values.